bin/generate_contributors.go

```diff
@@ -22,6 +22,15 @@ type IssueAuthorDetails struct {
 	url    string
 }
 
+func MapUsers(vs []*github.User, f func(*github.User) string) map[string]string {
```

Comment:
If this is to simulate a set of strings, you can do `map[string]struct{}` instead, as the values would be zero-width values.

Reply:
This whole thing is just so I can apply a function to each element in my list, and create a new list of the result. What I would want to do in Ruby down below is:
```
sourcegraphTeam.map{ |m| m.Login }
```

Reply:
If you don't care about uniqueness,

```
transformed := make([]string, 0, len(vs))
for _, v := range vs {
    transformed = append(transformed, f(v))
}

return transformed
```

Reply:
🎉  sweet! Thanks, that's what I needed.